src/docs/backup.dox

```diff
@@ -72,6 +72,22 @@ The following is a programmatic example of creating a backup:
 
 @snippet ex_all.c backup
 
+When logging is enabled, opening the backup cursor forces a log file switch.
+The reason is so that only data that was committed and visible at the time of
+the backup is available in the backup when that log file is included in the
+list of files. WiredTiger offers a mechanism to gather additional log files that
+may be created during the backup.
+
+Since backups can take a long time, it may be desirable to catch up at the
+end of a backup with the log files so that operations that occurred during
+backup can be recovered. WiredTiger provides the ability to open a duplicate
+backup cursor with the configuration \c target=log:. This secondary backup
+cursor will return the file names of all log files via \c dup_cursor->get_key().
+This secondary cursor must be closed explicitly prior to closing the parent
+backup cursor.
+
```

Comment:
Assuming the dup cursor returns "all" log files as stated, I think some further explanation of what the coder needs to do is in order.  For example if original backup returned log files 0,1,2,3,4,5 and all were copied, and the dup cursor returns 4,5,6,7, then are they obligated to copy 4 and 5?  I wouldn't think so because of the log file switch, but good to state that explicitly.

Reply:
Thanks. Yes they would not need to re-copy the overlap but there also is no harm in doing so. I'll update.